### PR TITLE
Upgrade deps

### PR DESCRIPTION
**Pull request by StratusFearMe21**
Simply upgrades all of this crate's dependencies to their latest versions

### CODE REVIEW

**Comment by StratusFearMe21** [comment]
P.S, this PR merges #33

File: Cargo.toml [hunk 1]
```diff
@@ -6,34 +6,35 @@ edition = "2021"
...
+serde = "1.0.160"
+toml = "0.7.3"
 colors-transform = "0.2.11"
-directories = "4.0.1"
+directories = "5.0.0"
 rustc-hash = "1.1.0"
 atomic_float = "0.1.0"
-egui_file = { git = "https://github.com/Kaydax/egui_file.git", rev = "7ac26fb" }
+egui_file = { git = "https://github.com/StratusFearMe21/egui_file.git", rev = "f8bd63b" }
+glidesort = "0.1.2"
```
Comment (arduano) [inline Cargo.toml:37]: Do we really need this? It's used to sort a single 256 item array each frame, `sort_unstable_by_key` should be enough for that.

**Review by arduano — CHANGES_REQUESTED** [review]
Everything else lgtm

**Review by arduano — APPROVED** [review]
Everything else lgtm

**Pull request merged**